pytorch_lightning/callbacks/quantization.py

```diff
@@ -126,9 +134,23 @@ def custom_trigger_last(trainer):
         quantize_on_fit_end: perform the quantization in `on_fit_end`.
             Note that once converted, the model cannot be put in training mode again.
 
+        disable_observers: disable fake-quantization modules' observers during the provided stages:
+
+            - ``'train'``: the observers will be disabled during training.
+            - ``'validate'``: the observers will be disabled during validating.
+              Note that we don't disable observers during the sanity check as the model hasn't been calibrated with
+              training data yet. After the sanity check, the fake-quantization modules are restored to initial states.
+            - ``'test'``: the observers will be disabled during testing.
+            - ``'predict'``: the observers will be disabled during predicting.
+
+            Note that we only handle observers belonging to fake-quantization modules. When ``qconfig`` is a ``str`` and
+            ``observer_type`` is ``'histogram'``, the observers won't belong to any fake-quantization modules and will
+            not be disabled.
+
     """
 
     OBSERVER_TYPES = ("histogram", "average")
+    DISABLE_OBSERVER_STAGES = ("train", "validate", "test", "predict")
```

Comment:
```suggestion
    OBSERVER_STAGES = ("train", "validate", "test", "predict")
```

Reply:
The `DISABLE_OBSERVER_STAGES` is changed to `OBSERVER_STAGES`.